### ISSUE TITLE
Fermipy-0.17.4/Python-2.7 Cannot handle lists in config.py

### ISSUE THREAD

**Issue by Alex-Broughton** [fermiPy/fermipy#379]
I have defined a region with in my config file as 
```
binning:

  binsz      : 0.08
  npix :
    - 625
    - 50
  binsperdec : 10
  coordsys   : 'GAL'
  proj : 'CAR'
```

The npix list keeps getting caught at the beginning during the read-in. I've found that it works in Python 3, but switching to Python 3 creates a plethora of other problems. Any way I can change cast_config() to get this to work?

Thanks!

```
(fermipy) [abrought@login-i16:~/h2sim] $python run_fermipy_full_sims.py 0
Warning in <TUnixSystem::SetDisplay>: DISPLAY not set, setting it to 10.240.2.34:0.0

WARNING: version mismatch between CFITSIO header (v3.43) and linked library (v3.41).

Traceback (most recent call last):
  File "run_fermipy_full_sims.py", line 134, in <module>
    main(sys.argv)
  File "run_fermipy_full_sims.py", line 56, in main
    gta = GTAnalysis('config.yaml',logging={'verbosity' : 3})
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/gtanalysis.py", line 271, in __init__
    **kwargs)
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/config.py", line 221, in __init__
    self.configure(config_dict, **kwargs)
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/config.py", line 229, in configure
    config = schema.create_config(config, **kwargs)
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/config.py", line 181, in create_config
    cast_config(config, self)
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/config.py", line 121, in cast_config
    cast_config(config[key], defaults[key])
  File "/data/homezvol2/abrought/miniconda/envs/fermipy/lib/python2.7/site-packages/fermipy/config.py", line 132, in cast_config
    config[key] = item_type(config[key])
TypeError: int() argument must be a string or a number, not 'list'
```

Upon using pdb, key = 'npix', item=[625, 50]

**Comment by Alex-Broughton** [comment]
Nevermind, fixed by installing fermipy version 0.20.0

**Comment by Alex-Broughton** [comment]
Fixed by installing fermipy version 0.20.0
